script/test_fixtures.sh

```diff
@@ -1,42 +1,61 @@
 #!/bin/bash
-set -ex
-
-make
+set -e
 
 test_folder() {
-    find "$1" -name "*_expected.rb" -maxdepth 1 | while read -r file
-    do
-        # shellcheck disable=SC2001
-        time ruby --disable=gems rubyfmt.rb "$(echo "$file" | sed s/expected/actual/)" > /tmp/out.rb
+    current_dir="$1"
+
+    ## Check that both files or neither exist
+    if [ -f "$current_dir/expected.rb" ] || [ -f "$current_dir/actual.rb" ]; then
+      [ -f "$current_dir/expected.rb" ] || (echo "$current_dir is missing the expected.rb file" && exit 1)
+      [ -f "$current_dir/actual.rb" ] || (echo "$current_dir is missing the actual.rb file" && exit 1)
```

Comment:
This currently checks that each file has a pair, in the main script. Do you like this or do you want a separate lint script that can check this?